src/figma/tokens.json

```
@@ -260,8 +260,8 @@
           "type": "color"
         },
         "red400": {
-          "value": "#E06470",
-          "description": "(HEX: #E06470)",
+          "value": "#FF5263",
```

This change should be reverted
```suggestion
          "value": "#E06470",
```